react-frontend/src/components/Home/cardlist.js

```diff
@@ -0,0 +1,172 @@
+import React, { useState, useEffect } from 'react';
+import InfoCard from './infocard';
+import { Col, Row } from 'antd';
+import '../../css/home.css';
+
+const descriptions = [
+  'The Digital Framework provides a high level vision and some expectations for how we work and what we deliver to enable digital ways of working.',
+  'Principles to guide the day-to-day work of BC Public Servants and contractors who are designing, developing, and delivering digital services.',
+  'Work is underway to renew policies towards information management and technology.',
+  'Find standards, policy, guides, and services to help you work more effectively in the digital age and deliver better services for citizens.',
+  'Connect with people working to improve digital services in British Columbia.',
+  'See examples of how teams are improving digital services for British Columbians.',
+];
+
+const imgLinks = [
+  'https://emasuvajac.com/wp-content/uploads/2017/06/DSC_2210a.jpg',
+  'https://theculturetrip.com/wp-content/uploads/2017/01/garibaldi-lake.jpg',
+  'https://www.naturallywood.com/sites/default/files/styles/inside_banner/public/coastalforest_04_0.jpg?itok=uF5QgK-Q',
+  'https://upload.wikimedia.org/wikipedia/commons/6/67/Vancouver%2C_BC%2C_Canada_landscape.jpg',
+  'https://i.cbc.ca/1.4486747.1515895578!/fileImage/httpImage/coastal-douglas-fir-ecosystem-southern-vancouver-island.jpg',
+  'https://upload.wikimedia.org/wikipedia/commons/5/5f/Howesound.JPG',
+];
+const linkTexts = [
+  'Read the framework',
+  'Read the principals',
+  'Read the policy',
+  'Find resources',
+  'Connect',
+  'Learn more',
+];
+function CardList() {
+  const [isMobile, setMobile] = useState(window.innerWidth <= 800);
+
+  React.useEffect(() => {
+    function handleResize() {
+      if (window.innerWidth >= 800) {
+        setMobile(false);
+      } else {
+        setMobile(true);
+      }
+    }
+    window.addEventListener('resize', handleResize);
+  });
+
+  if (isMobile) {
```

Comment:
I feel like this `if else` statement is probably not how we want to handle mobile views. It's been a while since I looked at the logic for it though.